runtime/ftplugin/qf_toc.vim

```diff
@@ -0,0 +1,23 @@
+function! s:setup_toc() abort
```

Comment:
We've already committed to forking man.vim so no need to avoid it there. And with this PR we admit that we're forking the help ftplugin. So I'd say let's not dance around it.